app/models/timer.rb

```diff
@@ -0,0 +1,31 @@
+class Timer < ActiveRecord::Base
+
+  belongs_to :player
+  belongs_to :game
+
+  def initialize
```

Comment:
After researching this more, I think I wrote this `after_initialize callback` more accurately, yet it is causing 2 failing tests :(. 

```
1) Timer should start timer for current players
     Failure/Error: self.timer ||= Timer.new
     
     NoMethodError:
       undefined method `timer' for #<Timer:0x007f564f3edbe8>
       Did you mean?  time
                      time=
                      time?
     # ./app/models/timer.rb:9:in `add_timer'
     # ./spec/models/timer_spec.rb:7:in `block (2 levels) in <top (required)>'
     # ./spec/models/timer_spec.rb:24:in `block (2 levels) in <top (required)>'

  2) Timer should stop timer for current players
     Failure/Error: self.timer ||= Timer.new
     
     NoMethodError:
       undefined method `timer' for #<Timer:0x007f564d663fa0>
       Did you mean?  time
                      time=
                      time?
     # ./app/models/timer.rb:9:in `add_timer'
     # ./spec/models/timer_spec.rb:7:in `block (2 levels) in <top (required)>'
     # ./spec/models/timer_spec.rb:29:in `block (2 levels) in <top (required)>'

Finished in 0.88573 seconds (files took 3.86 seconds to load)
34 examples, 2 failures

Failed examples:

rspec ./spec/models/timer_spec.rb:22 # Timer should start timer for current players
rspec ./spec/models/timer_spec.rb:27 # Timer should stop timer for current players

```

Here is the `after_initialize callback` I used - 

```
after_initialize :add_timer, :if => :new_record?

  def add_timer
    self.timer ||= Timer.new 
  end
```